deployments/pod-sriovdp.yaml

```diff
@@ -7,13 +7,13 @@ spec:
   - name: sriov-device-plugin 
     image: nfvpe/sriov-device-plugin:latest
     imagePullPolicy: IfNotPresent
-    command: [ "/bin/bash", "-c", "--" ]
-    args: [ "while true; do sleep 300000; done;" ]
+    command: [ "/bin/sh", "-c", "--" ]
+    args: [ "sriovdp --logtostderr 10; while true; do sleep 300000; done;" ]
```

Comment:
I think sriovdp will run Infinitely until it receives the ``sigCh`` signal, so perhaps remove the ``while`` loop in the command args?